src/styles/issues.module.scss

```diff
@@ -0,0 +1,10 @@
+.container {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	flex-wrap: nowrap;
+	margin-bottom: 1rem;
+	max-width: 750px;
+	width: 100%;
+	margin: auto;
```

Comment:
The `margin: auto` will override the `margin-bottom`.  I'm almost certain, but please check once.